src/api/services/adminServiceClient.ts

```diff
@@ -200,6 +207,19 @@ export const addKeyPair = async (
   return response.data;
 };
 
+export const updateKeyPair = async (
+  subscriptionId: string,
+  name: string,
+  disabled: boolean = false
+): Promise<KeyPairDTO> => {
+  const response = await adminServiceClient.post<KeyPairDTO>('/api/client_side_keypairs/update', {
+    subscription_id: subscriptionId,
+    disabled,
```

Comment:
The disabled flag in the update function here is good.  But the one above in the add function can be removed.  we no longer allow adding disabled keys.